Add spec for TwoPair

daelynj opened the pull request:
Check if the TwoPair service object returns true when given a hand
containing a two pair. Also check that a hand without a two pair returns
false.

daelynj committed:
Add spec for TwoPair

Check if the TwoPair service object returns true when given a hand
containing a two pair. Also check that a hand without a two pair returns
false.

jgayfer commented on spec/two_pair_spec.rb:
```diff
@@ -0,0 +1,33 @@
+      it { is_expected.to be false }
+    end
+  end
+end
```
Could be worth adding in a spec that takes in a hand with a single pair, not required though.

daelynj committed:
Add test to TwoPair for a single Pair

Add a test to the TwoPair spec to test a poker hand with one pair. This
should return false as it is not a TwoPair hand.
